GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -609,6 +609,13 @@ private void openWithDifftoolToolStripMenuItem_Click(object sender, EventArgs e)
 
             foreach (var itemWithParent in DiffFiles.SelectedItemsWithParent)
             {
+                if (itemWithParent.ParentRevision.Guid == GitRevision.CombinedDiffGuid)
```

Comment:
Should we do it for all artificial commits?

Reply:
worktree and index is specially handled in RevisionDiffProvider, they are OK